pkg/cli/alpha/internal/update/update.go

```diff
@@ -222,6 +222,29 @@ func runMakeTargets() {
 	}
 }
 
+// runMakeTargetsSilently is a helper function to run make with the targets necessary
+// to ensure all the necessary components are generated, formatted and linted.
+// This version runs silently without logging, suitable for best-effort execution when conflicts exist.
+func runMakeTargetsSilently() {
```

Comment:
I do not think that we should run silently
We need to suppress the error with force and raise a warning instead, see

 https://github.com/kubernetes-sigs/kubebuilder/pull/4992/files